src/averageAges.js

```diff
@@ -37,7 +40,14 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter(person => person.sex === 'f'
+    && people.some(children => person.name === children.mother))
+    : people.filter(person => person.sex === 'f');
```

Comment:
same here, use if and assign default value to women (array with all women who are mothers in it)